tsl/test/sql/include/compression_policy.sql

```diff
@@ -0,0 +1,142 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
```

Comment:
This file appears to be identical/similar to compression_bgw.sql. Do we need this new file? Where is it included?